analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -29,6 +29,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.msbuild" Version="3.1.2">
```

Comment:
in [AutoScan](https://github.com/SonarSource/sonar-dotnet-autoscan/pull/213/files#diff-60390b2526c39a5f9fdbf541ef1ae0f0fc11294d6c4b93deb7118528e02e25db) I see that `<PackageReference Include="coverlet.collector" Version="3.1.0">` is necessary as well - here it is not?

Reply:
Yeap! I remembered yesterday evening when I start having 0 coverage only in the CI! 😅